cgi-bin/LJ/S2.pm

```diff
@@ -2645,8 +2645,11 @@ sub sitescheme_secs_to_iso {
     # if opts has a true tz key, get the remote user's timezone if possible
     my $s2_datetime = $opts{tz} ? DateTime_tz( $secs, $remote ) : undef;
     my $has_tz = defined $s2_datetime ? "(local)" : "UTC";
-    # if timezone execution failed, use GMT
-    $s2_datetime = DateTime_unix( $secs ) unless defined $s2_datetime;
+    # if timezone execution failed, use GMT (but don't display timezone)
```

Comment:
Oh, I thought the first assignment to $has_tz was only true if $opts{tz} was true... I was thinking that in the undefined timezone case, it should print UTC if $opts{tz} was true, and nothing if it was false.